### PR TITLE
[DO NOT MERGE] Version 7.1.0

### PR DESCRIPTION
**Pull request by erikd**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ services:
         max-file: "10"
 
   cardano-node:
-    image: inputoutput/cardano-node:1.23.0
+    image: inputoutput/cardano-node:1.24.1
```
Comment (rhyslbw) [inline docker-compose.yml:27]: `1.24.2`

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (erikd) [reply docker-compose.yml]: Fixed.

**Comment by ksaric** [comment]
@erikd Can you please leave the commit `3cf868ec8c06265e4b670ac737af640d716e5ef7` separate and not rewrite Git history? For now, I'm depending on that commit and once you have things ready we can sync up and I can target your merge commit.

**Comment by erikd** [comment]
If no further issues are found and this does indeed become the `7.1.0` release, this commit will remain. If an issue is found and it needs to be corrected, this commit will be replaced by the fix commit.

**Comment by ksaric** [comment]
> If no further issues are found and this does indeed become the `7.1.0` release, this commit will remain. If an issue is found and it needs to be corrected, this commit will be replaced by the fix commit.

Yes, but could you just temporarily add a new commit and squash them when it's finalized?

**Pull request closed without merging**